### PR TITLE
Reorganize manual

### PR DESCRIPTION
**Pull request by fingolfin**
Grouping things differently in a way that makes more sense to me personally.

### CODE REVIEW

File: docs/src/classes.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+of class types. The results are correct for all
+possible combinations of parameters except possibly for those where the
+additionally returned exceptions apply. Those consist of multivariate
+polynomials with coefficients in a rational function field and are
+satisfied if the evaluation of this polynomial is an integer.
+
+
+```@docs
+ortho2norm
+ortho2scalar
```
Comment (fingolfin) [inline docs/src/classes.md:31]: Should we rename those to norm resp scalar_product ?

**Comment by fingolfin** [comment]
@SoongNoonien any objections or concerns? I mainly moved things around and added TODO comments (I'll resolve a few later but I wanted to get the basic structure right first)

File: docs/src/classes.md (hunk 1 above)
Comment (SoongNoonien) [reply docs/src/classes.md:31]: Yes, that's a good idea. As we don't have the old index based methods for the characters anymore we should get any conflicts.

**Comment by SoongNoonien** [comment]
> @SoongNoonien any objections or concerns?

No, none. The structure of GenericCharacterTables has been deviating form the docs for quite some time now.

**Pull request merged**